scripts/build/extend_version_file.py

```diff
@@ -35,56 +35,71 @@ def extend_version_file(repository_root, version_file):
     with open(version_file) as v_file:
         version_json_data = json.load(v_file)
 
-    git_hash = ''
-    try:
-        git_hash_cmd = ['git', 'rev-parse', 'HEAD']
-        git_hash = run_cmd(git_hash_cmd, repository_root)
-        git_hash = git_hash.rstrip()
-    except subprocess.CalledProcessError:
-        LOG.exception('Failed to get last commit hash.')
-    except OSError:
-        LOG.exception('Failed to run command: %s', ' '.join(git_hash_cmd))
-        sys.exit(1)
-
     version = version_json_data['version']
     version_string = str(version['major'])
     if int(version['minor']) != 0 or int(version['revision']) != 0:
         version_string += ".{0}".format(version['minor'])
     if int(version['revision']) != 0:
         version_string += ".{0}".format(version['revision'])
 
-    git_describe = git_describe_dirty = version_string
-    try:
-        # The tag only (vX.Y.Z)
-        git_describe_cmd = ['git', 'describe', '--always', '--tags',
-                            '--abbrev=0']
-        git_describe = run_cmd(git_describe_cmd, repository_root)
-        git_describe = git_describe.rstrip()
-
-        # The full dirty hash (vX.Y.Z-n-gabcdef0-tainted)
-        git_describe_cmd = ['git', 'describe', '--always', '--tags',
-                            '--dirty=-tainted']
-        git_describe_dirty = run_cmd(git_describe_cmd, repository_root)
-        git_describe_dirty = git_describe_dirty.rstrip()
-
-        # Always replace the Git tag with the manually configured version
-        # information -- but keep the "tainted" flag (if the working directory
-        # differs from a commit) information visible, along with the
-        # abbreviated command hash.
-        # (This makes the tag compiled into the package improper, but this
-        # is a design decision of the developer team!)
-        git_tag = version_string
-        git_tag_dirty = git_describe_dirty.replace(git_describe,
-                                                   version_string)
-    except subprocess.CalledProcessError:
-        LOG.exception('Failed to get last commit describe.')
-    except OSError:
-        LOG.exception('Failed to run command: %s', ' '.join(git_describe_cmd))
-        sys.exit(1)
-
-    version_json_data['git_hash'] = git_hash
-    version_json_data['git_describe'] = {'tag': git_tag,
-                                         'dirty': git_tag_dirty}
+    LOG.info("This is CodeChecker v%s", version_string)
+
+    if os.path.exists(os.path.join(repository_root, '.git')):
```

Comment:
Could you move the part where we get the git information into a separate function.